sagemaker/launch/launch_partition.py

```diff
@@ -45,40 +44,43 @@ def run_job(input_args, image):
     sagemaker_session = sagemaker.Session(boto3.Session(region_name=region))
 
     skip_partitioning_str = "true" if input_args.skip_partitioning else "false"
-    params = {"graph-data-s3": graph_data_s3,
-              "metadata-filename": metadata_filename,
-              "num-parts": num_parts,
-              "output-data-s3": output_data_s3,
-              "skip-partitioning": skip_partitioning_str,
-              "log-level": input_args.log_level,
-              "partition-algorithm": input_args.partition_algorithm,}
-
-    print(f"Parameters {params}")
+    arguments = [
+        "--graph-data-s3", graph_data_s3,
+        "--metadata-filename", metadata_filename,
+        "--num-parts", num_parts,
+        "--output-data-s3", output_data_s3,
+        "--skip-partitioning", skip_partitioning_str,
+        "--log-level", input_args.log_level,
+        "--partition-algorithm", input_args.partition_algorithm,
+    ]
+    arguments = [str(x) for x in arguments]
+
+    print(f"Parameters {arguments}")
     if input_args.sm_estimator_parameters:
         print(f"SageMaker Estimator parameters: '{input_args.sm_estimator_parameters}'")
 
     estimator_kwargs = parse_estimator_kwargs(input_args.sm_estimator_parameters)
 
-    est = PyTorch(
-        disable_profiler=True,
-        debugger_hook_config=False,
-        entry_point=os.path.basename(entry_point),
-        source_dir=os.path.dirname(entry_point),
+    script_processor = ScriptProcessor(
         image_uri=image,
         role=role,
-        instance_count=instance_count,
+        instance_count=int(instance_count),
```

Comment:
Shouldn't instance_count be a int? We can define the type of --instance-count

Reply:
Yeah actually that's already the case, so I'll remove this : https://github.com/awslabs/graphstorm/blob/main/sagemaker/launch/common_parser.py#L33